primitives/src/crh/bowe_hopwood/mod.rs

```diff
@@ -6,13 +6,17 @@ use std::{
     marker::PhantomData,
 };
 
-use super::pedersen::{PedersenCRH, PedersenWindow};
 use crate::crh::FixedLengthCRH;
-use algebra::{biginteger::BigInteger, fields::PrimeField, groups::Group};
+use algebra::{Group, fields::{PrimeField, FpParameters}};
 use serde::{Deserialize, Serialize};
 
 pub const CHUNK_SIZE: usize = 3;
 
+pub trait PedersenWindow: Clone {
+    const WINDOW_SIZE: usize;
+    const NUM_WINDOWS: usize;
+}
```

Comment:
Let us document here the definition of `WINDOW_SIZE` and `NUM_WINDOWS`, including security conditions.